notebooks/week2/05.log_and_register_fe_model.py

```
@@ -0,0 +1,192 @@
+# Databricks notebook source
+# MAGIC %pip install ../wine_quality-0.0.1-py3-none-any.whl
+
+# COMMAND ----------
+
+dbutils.library.restartPython()
+
+# COMMAND ----------
+
+import mlflow
```

Ruff

Module level import not at top of file

(E402)